geniza/corpus/management/commands/import_data.py

```diff
@@ -297,6 +298,68 @@ def add_document_language(self, doc, row):
             doc.language_note = "\n".join(notes_list)
             doc.save()
 
+    def parse_notes(self, doc, row):
+        if "PGPID" in row.notes:
+            old_pgpids = row.notes.split(": ")[-1]
+            doc.old_pgpids = [int(old_pgpid) for old_pgpid in old_pgpids.split(", ")]
+
+        IGNORE = [
+            "India",
+            "India; unsure of date entered by Geniza Lab team (historical, before 2015)",
+            "India: addendum to IB IV",
+            "Rustow, Lost Archive",
+            "DISAGGREGATE",
+        ]
+
+        notes_to_set = []
+        if row.notes not in IGNORE:
+            notes_to_set.append(row.notes)
+
+        ## TECHNICAL NOTES
+        # Unique values:
+        """
+            FGP stub
+            scanned in drive
+            ?scanned in drive
+            Transcription needs to be uploaded*
+            Transcription needs to be uploaded
+            scanned in drive (TRANSCRIPTION)
+            scanned in drive (TRANSLATION)
+            not in Gil
+            Scanned in drive
+            scanned in drive 
+            scanned in drive (TRANSCRIPTION + TRANSLATION)
+            scanned in drive (TRANSCRIPTION & TRANSLATION)
+        """
```

Comment:
Simpler to make this a dict lookup of old text to new text, at least for the simple cases?

Reply:
mmmmmmm yeah I don't know. It works as it is, but I agree that a lookup would be just as good 🤷‍♀️

Reply:
you're right! if it's working and we don't need to mess with it, just get it done for tomorrow